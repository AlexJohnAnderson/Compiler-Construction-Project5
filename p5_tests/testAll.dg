int func(){
    return 10;
    output(func());
}
void main(){
    output(main());
}