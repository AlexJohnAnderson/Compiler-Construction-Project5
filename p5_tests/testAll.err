FATAL [7,8]-[7,9]: Non-bool expression used as an if condition
FATAL [14,16]-[14,18]: Non-bool expression used as a loop condition
FATAL [23,11]-[23,12]: Non-bool expression used as a loop condition
FATAL [29,5]-[29,18]: Attempt to output void
FATAL [30,8]-[30,14]: Non-bool expression used as an if condition
Type Analysis Failed
