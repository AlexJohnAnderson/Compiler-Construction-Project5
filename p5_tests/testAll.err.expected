FATAL [6,12]-[6,18]: Attempt to output void
Type Analysis Failed
